apis/python/tests/test_type_diversity.py

```diff
@@ -199,16 +201,14 @@ def cmp_dtype(series, tdb: tiledb.Attr) -> bool:
                 assert cmp_dtype(df[k], arr.schema.attr(attr_idx[k]))
 
 
-# TODO: re-enable when #45 is resolved
-@pytest.mark.skip(reason="Fails: filed as issue #45")
 def test_from_anndata_annotations_empty(tmp_path):
     """
     Validate correct conversion with an empty (index-only) obs/var
     """
     n_obs = 100
     n_var = 10
 
-    # AnnData requires a string index. TileDB does not support UTF8, so indices must be ASCII.
+    # AnnData requires a string index. TileDB does not support UTF8, so use ASCII.
```

Comment:
@ihnorton what's the current status on this?

Reply:
Dimensions only support ASCII, attributes support either ASCII or UTF-8.